Jenkinsfile

```diff
@@ -522,7 +522,12 @@ pipeline {
             }
         }
         stage('End-to-End Performance') {
-            agent { label 'benchmark' }
+            agent {
+                docker {
+                    image 'noisepage:focal'
+                    args '--cap-add sys_ptrace -v /jenkins/ccache:/home/jenkins/.ccache'
+                }
+            }
```

Comment:
Why are we changing the performance test machine?

Reply:
Do we need to keep track of performance of this pipeline?

Reply:
What do you mean by `keep track of performance`? I guess my point is more about that you're changing the machine for the `End-to-End Performance` stage, which is not the stage we're adding.

Reply:
Oh yes, this was a good catch.